doc/Language/variables.pod6

```diff
@@ -222,7 +221,7 @@ old variable as we did with C<$*dynamic2>.
 The dynamic variables differ from other variable types in that referring
 to an undeclared dynamic variable is not a compile time error but a
 runtime L<failure|/type/Failure>, so a dynamic variable can be used
-undeclared as long as it is checked for definedness or used in a
```

Comment:
How about:
`Dynamic variables are checked at runtime, as opposed to compile time. If your code refers to an undeclared dynamic variable at runtime, you will get a runtime L<failure|/type/Failure>, unless the variable is used in a boolean context or tested for definedness before being used for something else.`